package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "Where programmers share ideas and help each other grow",
   "engines": {
-    "yarn": "1.21.x"
+    "yarn": ">=1.21.x"
```

Comment:
@nickytonline Is this ok? We had a problem with our Docker builds, as they ended up pulling in Yarn 1.22. If you're unhappy with this change I'll make Docker pull a 1.21.x version.

Reply:
I think we need to change that to disallow Yarn 2 as I think the code doesn't work with it, other than that, yep!

See https://github.com/thepracticaldev/dev.to/issues/5740

Reply:
Please merge this. Docker pulls yarn 1.22.4 and docker build fails.

Reply:
@nickytonline I had a quick look at the docs and it seems there'll be no implicit updates to Yarn 2 and Yarn 2 is not recommended to ever be installed globally. 

If you want to change this please feel free to do so, I'm not entirely sure what the correct syntax here is. In the context of Docker we install 1.22, so if you feel more comfortable with "1.22.x" that's good enough for this use-case.

Reply:
> @nickytonline Is this ok? We had a problem with our Docker builds, as they ended up pulling in Yarn 1.22. If you're unhappy with this change I'll make Docker pull a 1.21.x version.

This seems fine to me and we have a .yarnrc file which should always force 1.21 to be used. @rhymes added this in Feb. 5, https://github.com/thepracticaldev

Reply:
> > @nickytonline Is this ok? We had a problem with our Docker builds, as they ended up pulling in Yarn 1.22. If you're unhappy with this change I'll make Docker pull a 1.21.x version.
> 
> This seems fine to me and we have a .yarnrc file which should always force 1.21 to be used. @rhymes added this in Feb. 5, https://github.com/thepracticaldev

Although I have .yarnrc in my project, docker will install 1.22.4